src/use-cases/appointment.tsx

```diff
@@ -0,0 +1,85 @@
+import { eq, and } from "drizzle-orm";
+import { db } from "~/db";
+import { appointments } from "~/db/schema";
+import { 
+    type GetAppointmentInput, 
+    type GetAppointmentResult, 
+    type ListAppointmentsInput,
+    type ListAppointmentsItem,
+    type ListAppointmentsResult,
+    type UpdateAppointmentInput,
+    type UpdateAppointmentResult,
+    type CreateAppointmentInput,
+    type CreateAppointmentResult
+} from "~/entities/appointment";
+
+export async function getAppointment(input: GetAppointmentInput): Promise<GetAppointmentResult> {
+    const result = await db
+        .select()
+        .from(appointments)
+        .where(
+            eq(appointments.appointmentId, input.appointmentId)
+        );
+    const appointment = result.at(0);
+    if (appointment) {
+        return appointment
+    } else {
+        throw new Error("No journal found")
+    }
+}
+
+export async function updateAppointmentItem(input: UpdateAppointmentInput): Promise<UpdateAppointmentResult> {
+    const result = await db
+        .select()
+        .from(appointments)
+        .where(
+            and(
+                eq(appointments.appointmentDate, input.appointmentDate),
+                eq(appointments.appointmentId, input.appointmentId),
+                eq(appointments.notes, input.notes),
+            )
+        );
+    const appointment = result.at(0);
+    if (appointment) {
+        return appointment
+    } else {
+        throw new Error("No journal found")
+    }
+}
```

Comment:
The function should be named `updateAppointment`.

Also the function doesn't update anything it just selects from the db.